packages/interbit/src/scripts/hoist.js

```diff
@@ -46,8 +47,7 @@ const hoistPackages = async ({ appLocation, covenantConfig }) => {
   }
   console.log('interbit-hoist: Writing new dependencies', hoistedPackageJson)
 
-  const writeData = `${JSON.stringify(hoistedPackageJson, null, 4)}\r\n`
-  fs.writeFileSync(appPackageJsonPath, writeData, 'utf8')
+  writeJsonFile(appPackageJsonPath, hoistedPackageJson)
 
```

Comment:
I notice that some json files were indented 2 chars and some 4. Does your refactoring ensure consistent indent?.

Reply:
If I didn't miss a JSON write, it should have. ;)